cloudferrylib/os/actions/check_bandwidth.py

```diff
@@ -29,6 +32,12 @@
 
 class CheckBandwidth(action.Action):
     def run(self, **kwargs):
+        runner = remote_runner.RemoteRunner(self.cloud.cloud_config.cloud.host,
+                                            self.cloud.cloud_config.cloud.ssh_user)
+
+        temp_dir_name = tempfile.mkdtemp(prefix=self.__class__.__name__)
```

Comment:
According to [documentation](https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp):
> tempfile.mkdtemp()
> Creates a temporary directory <...>
> <...>
> The user of mkdtemp() is responsible for deleting the temporary directory and its contents when done with it.
This doesn't seem right, we should not create anything locally. It should generate remote name for the target host without touching anything locally